docs/api-testing/schedule-test.md

```diff
@@ -12,35 +12,29 @@ The **Schedule** tool, accessible from within each test, allows you to schedule
 ## What You'll Need
 
 * A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
-* An existing API Testing Project and Test. For details on how to create them, see the [Quickstart](/api-testing/quickstart/) guide.
+* An existing, published API Test. For details on how to create one, see the [Quickstart](/api-testing/quickstart/).
 
 
-## Publish the Working Copy
+## Creating a New Test Schedule
 
-Our API Testing interface has a unique _Working Copy_ / _Published Copy_ system that allows you to edit a test without affecting the live, currently active version.
+:::caution Required Step
+Do not proceed until you've [published the working copy of your test](/api-testing/quickstart/#publish-your-test).
+:::
 
-1. Once you've finish editing your test, navigate to the **Unpublished changes** section, where it states that your test is currently unpublished.<br/>
-<img src={useBaseUrl('img/api-fortress/2021/02/unpublished.png')} alt="Unpublished Test" width="200" />
-2. Hover your mouse over the section to reveal the **Clear** and **Publish** buttons.<br/>
-<img src={useBaseUrl('img/api-fortress/2021/02/publishButton.png')} alt="Publish Test Button" width="200"/>
-3. Click the **Publish** button to publish your working copy.<br/>
-<img src={useBaseUrl('img/api-fortress/2021/02/published.png')} alt="Published Test" width="200"/>
+1. Click the **Schedule** tab to access the Scheduler.<br/><img src={useBaseUrl('img/api-fortress/2021/02/scheduleTab.png')} alt="Schedule Tab"/>
```

Comment:
@kimsauce from where? Schedule tab is available within the test, it's not clear where the user should be for clicking that tab

Reply:
good catch. I will revise this.